content/essentials/understanding-jsx.mdx

```diff
@@ -0,0 +1,105 @@
+import { Aside } from "~/components/configurable/Aside";
+
+# Understanding JSX
+
+JSX is a syntax extension for JavaScript. 
+It allows you to write HTML-like code inside your JavaScript file which keeps your rendering logic and content in the same place. 
+This provides a concise and readable way to create and represent components.
+
+## How Solid uses JSX
+
+Solid was designed to align closely with HTML standards. 
+It offers a distinct advantage: copy/paste solutions from resources like Stack Overflow, and direct usage of templates from design tools.
```

Comment:
I've reread this several times and I'm struggling to get a grasp of what you were trying to say. I would suggest rewriting this sentence. 

> It offers a distinct advantage, however: to copy/paste solutions from resources like Stack Overflow; and to allow direct usage of templates from design tools.

Something like that? I'm not sure.